docs/content/contribute/index.md

```diff
@@ -1,48 +1,109 @@
-# Contributing to the Docs
+# 🚀 Contributing to the Docs
 
-We welcome contributions to our documentation! If you would like to contribute, please follow the steps below.
+Welcome to our documentation contribution guide! We're excited to have you here. This guide will help you get started with contributing to our documentation. Let's make our docs better together! 💪
 
-## Setting up the Docs
+<div align="center">
 
-1. Clone the repository:
+![Documentation Contributors](https://img.shields.io/github/contributors/Significant-Gravitas/AutoGPT?style=for-the-badge)
+![Pull Requests Welcome](https://img.shields.io/badge/PRs-welcome-brightgreen.svg?style=for-the-badge)
+![Documentation Build Status](https://img.shields.io/badge/docs-passing-success?style=for-the-badge)
 
-    ```shell
-    git clone github.com/Significant-Gravitas/AutoGPT.git
-    ```
+</div>
 
-1. Install the dependencies:
+## 🛠️ Setting up the Docs
 
-    ```shell
-    python -m pip install -r docs/requirements.txt
-    ```
+### Prerequisites
 
-    or
+- Git installed on your machine
+- Python 3.8 or higher
+- A text editor of your choice
 
-    ```shell
-    python3 -m pip install -r docs/requirements.txt
-    ```
+### Step-by-Step Setup Guide
 
-1. Start iterating using mkdocs' live server:
+1. **Clone the Repository** 📥
+   ```shell
+   git clone github.com/Significant-Gravitas/AutoGPT.git
+   ```
 
-    ```shell
-    mkdocs serve
-    ```
+2. **Install Dependencies** 📚
+   ```shell
+   python -m pip install -r docs/requirements.txt
+   ```
+   or
+   ```shell
+   python3 -m pip install -r docs/requirements.txt
+   ```
 
-1. Open your browser and navigate to `http://127.0.0.1:8000`.
+3. **Start the Development Server** 🔥
+   ```shell
+   mkdocs serve
+   ```
```

Comment:
You can also pass `-a localhost:8392` to run it on a different port so it doesn't conflict with supabase when running the full stack